en/docs/try-out/account-and-transaction-flow.md

```diff
@@ -228,50 +228,40 @@ The AISP application redirects the bank customer to authenticate and approve/den
 
     ``` tab="Format"
     {
-      "kid": "<CERTIFICATE_FINGERPRINT>",
-      "alg": "<SUPPORTED_ALGORITHM>",
+      "kid": "2MI9XSKi6ddxCbWg2rhDNtUlxJc",
```

Comment:
let's use ""kid": "<CERTIFICATE_FINGERPRINT>",
      "alg": "<SUPPORTED_ALGORITHM>","
as this section shows the format

Reply:
Fix review comments